OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -476,32 +481,31 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </note>
                                 </item>
                                 <item xml:id="e4">
-                                    <locus target="#26"/>
-                                    <note>A green thread for navigating is inserted on the upper corner of the folio.</note> <!-- move this to binding -->
-                                </item>
-                                <item xml:id="e5">
                                     <note>Text written on erasure, e.g. <locus target="#12ra #39rb #44rb"/>.</note>
                                 </item>
-                                <item xml:id="e6">                         
+                                <item xml:id="e5">                         
                                     <note>Erasures marked with thin black lines, e.g. <locus target="#73vb #74va"/>.</note>
                                 </item>
                             </list>
                         </additions>
                         
                         <bindingDesc>
                             <binding contemporary="true">
-                                <decoNote xml:id="b1">Two wooden boards covered with reddish-brown tooled leather.
-                                    <!-- two layers of leather cover!! -->
+                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards covered with reddish-brown <material key="leather"/>.
+                                    Presence of a reddish-brown tooled <material key="leather"/> overback in correspondence of the spine.
                                 </decoNote>
                                 <decoNote xml:id="b2" type="bindingMaterial">
                                     <material key="wood"/>       
                                     <material key="leather"/>     
                                 </decoNote>
-                                <decoNote xml:id="b3" type="SewingStations">4</decoNote>
-                                <decoNote xml:id="b5">Blind tooled ornament on the front and back covers (circles, double circles, double straight lines).</decoNote>
+                                <decoNote xml:id="b3" type="Cover" color="reddish-brown">Blind-tooling decoration on the overback:
+                                    double lines, double circles.</decoNote>
+                                <decoNote xml:id="b4" type="SewingStations">4</decoNote>
+                                <decoNote xml:id="b5" type="Spine">Two pairs of chains of the endband stitches are visible at the spine cover, close to the head and tail.</decoNote>
```

Comment:
Two and pairs is 4. "The stitches of a pair of endband chains are sewn into the leather of the spine."?

Reply:
@Gnisci 
I guess there are two chains at the head and two chains at the tail.
The endbands probably were sewn to the head and tail edges and you can see the stichtes at the spine. (unfortunately I cannot check on the images...)